pyproject.toml

```diff
@@ -80,13 +75,29 @@ norecursedirs = ".git .tox _resource .mypy_cache build dist docs"
 DJANGO_SETTINGS_MODULE = "tests.test_app.library.settings"
 FAIL_INVALID_TEMPLATE_VARS = 1
 
-[tool.flake8]
-max-complexity = 10
-ignore = ["C901", "W504", "W503", "E231", "E203"]
-exclude = [".git", ".tox", "_resource", "migrations", "build", "dist", "docs"]
-max-line-length = 120
-show-source = true
-statistics = false
+[tool.ruff]
+target-version = "py38"
+line-length = 120
+
+[tool.ruff.lint]
+select = [
+    "E",    # PEP8 errors
+    "W",    # PEP8 warnings
+    "F",    # PyFlakes
+    "I",    # isort
+    "C",    # mccabe
+    "T201", # flake8-print
+]
+ignore = [
+    "E501", # line too long
+]
+
+[tool.ruff.lint.isort]
+known-first-party = ["jazzmin"]
+known-local-folder = ["tests"]
+
+[tool.ruff.lint.mccabe]
+max-complexity = 12
```

Comment:
10 is a more sensible number, I find that it catches things that are genuinely in need of complexity reduction

Reply:
Updated 👍🏻